recipients_app/ios/Runner/Runner.entitlements

```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
-<dict/>
+<dict>
+	<key>aps-environment</key>
+	<string>development</string>
```

Comment:
The aps-environment seems to be set automatically by Xcode depending on the used provisioning profile.

```
Xcode sets the value of the entitlement based on your app's current provisioning profile. For example, if you're using a development provisioning profile, Xcode sets the value to development. Production provisioning profile and [Prerelease Versions and Beta Testers](https://developer.apple.com/documentation/appstoreconnectapi/prerelease_versions_and_beta_testers) use production. These default settings can be modified. The development environment is also referred to as the sandbox environment.
```

see https://developer.apple.com/documentation/bundleresources/entitlements/aps-environment